packages/scatterplot/src/ScatterPlotCanvas.js

```diff
@@ -198,10 +199,15 @@ class ScatterPlotCanvas extends Component {
         })
 
         points.forEach(point => {
-            this.ctx.beginPath()
-            this.ctx.arc(point.x, point.y, getSymbolSize(point.data) / 2, 0, 2 * Math.PI)
-            this.ctx.fillStyle = getColor(point.data)
-            this.ctx.fill()
+            if (symbol) {
```

Comment:
For the canvas implementation I propose something like this, let me know what you think.

Reply:
We could use the same approach as for the SVG implementation, setting a default symbol function and making it required.
Also `this.ctx.fill()` should be removed as it's part of the symbol rendering function.